lib/src/util/console_log_utils.dart

```diff
@@ -94,12 +91,12 @@ T spyOnConsoleLogs<T>(
 /// See also: [printConsoleLogs], [spyOnConsoleLogs]
 void Function() startSpyingOnConsoleLogs({
   ConsoleConfig configuration = ConsoleConfig.all,
-  @required void Function(String) onLog,
+  required void Function(String) onLog,
 }) {
   final logTypeToCapture = configuration.logType == 'all' ? ConsoleConfig.types : [configuration.logType];
   final consoleRefs = <String, /* JavascriptFunction */ dynamic>{};
   final consolePropertyDescriptors = <String, dynamic>{};
-  final _console = getProperty(window, 'console');
+  final _console = getProperty(window, 'console') as Object;
```

Comment:
Looks like `getProperty` accepts a generic, so instead of casting you should be able to do 

```suggestion
  final _console = getProperty<Console>(window, 'console');
```

Unless you're explicitly trying to loosen the type to `Object`, in which case you can use that as the generic.

Reply:
FYI getProperty's generic was only added in newer Dart SDKs (I think Dart 2.19, but am not sure), so it may not work if we're running on 2.19.

#nit Since we're interacting with this as a JS object, typing it as `Console` instead of `Object` unnecessary.